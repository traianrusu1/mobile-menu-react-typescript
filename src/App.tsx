import React from 'react';
import styles from './App.module.scss';
import { Layout } from './components';

const App: React.FC = () => {
  return (
    <>
      <Layout>
        <div className={styles.myApp}>THIS IS MY APP</div>
      </Layout>
    </>
  );
};

export default App;
