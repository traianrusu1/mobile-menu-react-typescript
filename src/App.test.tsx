import React from 'react';
import { render } from '@testing-library/react';
import App from './App';

describe('<App />', () => {
  test('renders', () => {
    const { container } = render(<App />);
    expect(container).toMatchSnapshot();
  });
});
