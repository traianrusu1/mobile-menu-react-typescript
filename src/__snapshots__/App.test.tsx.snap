// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<App /> renders 1`] = `
<div>
  <div
    class="layout"
  >
    <div
      class="toolbar"
    >
      Toolbar
      
    </div>
    <div
      class="myApp"
    >
      THIS IS MY APP
    </div>
  </div>
</div>
`;
